sklearn/ensemble/tests/test_gradient_boosting.py

```diff
@@ -849,6 +849,34 @@ def test_warm_start_oob():
                                   est.oob_improvement_[:100])
 
 
+def test_warm_start_sparse():
+    # Test that all sparse matrix types are supported
+    X, y = datasets.make_hastie_10_2(n_samples=100, random_state=1)
+    sparse_matrix_type = [csr_matrix, csc_matrix, coo_matrix]
+    for Cls in [GradientBoostingRegressor, GradientBoostingClassifier]:
```

Comment:
use `cls` (small 'c') rather than `Cls`, this is a variable, even if its value is a class.

Reply:
I wanted to keep the same style than all the other `warm_start` tests. I really don't like to have different way of writing it. Shall I change all of them, change only this one, or let it as it is.